oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/response/search/SearchHits.java

```diff
@@ -48,7 +48,7 @@ public Iterator<SearchHit> iterator() {
         return getHits().iterator();
     }
 
-    static final class TotalDeserializer extends JsonDeserializer<Integer> {
+    public static final class TotalDeserializer extends JsonDeserializer<Integer> {
```

Comment:
Is this an unexpected change? Why `public`?

Reply:
As an inner class, it should be accessible outside. Where do you use?

Reply:
In SkyWalkingFeature, but I'm hesitating on whether to keep this class. I'll take a look after the CI for my repository finishes running.

Reply:
OK. Let's continue the discussion after you have more conclusions.